icecreamshop/src/main/java/com/summerschool/icecreamshop/controller/CategoryController.java

```diff
@@ -44,5 +48,15 @@ public CategoryController(CategoryService categoryService, ModelMapper modelMapp
 
         return ResponseEntity.ok(modelMapper.map(category, CategoryDTO.class));
     }
+    @PatchMapping(path = "/{categoryId}")
+    public ResponseEntity<CategoryDTO> update(@PathVariable("categoryId") Long id, @RequestBody @Valid CategoryDTO categoryDTO){
+        categoryService.get(id)
+                .orElseThrow(()-> new ResponseStatusException((HttpStatus.NOT_FOUND), CATEGORY_NOT_FOUND));
+
+        categoryDTO.setId(id);
+
+        Category updateCategory = categoryService.add(modelMapper.map(categoryDTO,Category.class));
```

Comment:
categoryService.patch(...) , trebuie implementata metoda patch() in serviciu pentru a corespunde cu tipul requestului

Reply:
adauga te rog la pull request si cateva screenshoturi cu call-urile facute in Postman pentru fiecare tip de response pe care il poti primi